json-logs/samples/api/conversations.requestSharedInvite.approve.json

```diff
@@ -0,0 +1,4 @@
+{
+  "ok": true,
+  "invite_id": "I1234"
```

Comment:
Like other APIs, `"error": "",` and `"response_metadata": { ... }` (response_metadata does not always exist tho) should exist for error patterns. That said, if you need only these properties so far, it's okay to go ahead with it.